src/Components/Pages/TransferPage.tsx

```diff
@@ -309,7 +321,7 @@ const TransferPage = (): JSX.Element => {
               setWalletType('Ethereum');
             }}
           >
-            Bridge to wCFG
+            Bridge wCFG
```

Comment:
I think `Bridge to wCFG` makes more sense but I will ask Cassidy or Lucas.

Reply:
"Bridge to" got me confused, because it sets the target perspective. I was coming from the assumption: "I have CFG/wCFG that I want to bridge (to somewhere else)

Reply:
Used `Get` as per Cassidy